hivelvet-backend/app/src/Actions/Labels/Delete.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Hivelvet open source platform - https://riadvice.tn/
+ *
+ * Copyright (c) 2022 RIADVICE SUARL and by respective authors (see below).
+ *
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation; either version 3.0 of the License, or (at your option) any later
+ * version.
+ *
+ * Hivelvet is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with Hivelvet; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Actions\Labels;
+
+use Actions\Delete as DeleteAction;
+use Actions\RequirePrivilegeTrait;
+use Enum\ResponseCode;
+use Models\Label;
+/**
+ * Class Delete.
+ */
+class Delete extends DeleteAction {
+    use RequirePrivilegeTrait;  
+    public function execute($f3, $params): void
+    {
+        $label = new Label(); 
```

Comment:
Normally the inheritance should be enough.

Reply:
There is a problem with the `Actions\Delete->execute` where the `$resultCode` is declared and holds the response status code but never been used and therefore it will always return the default status `200 OK` plus it does log but does not render any data.
 
I thought about keeping this PR consistent with the rest of the delete actions logic and discuss the `Actions\Delete->execute`  in the next meeting before making any changes.